doc/DeveloperGuide.md

```diff
@@ -58,8 +58,11 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | new user | see usage instructions | refer to instructions when I forget how to use the App
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
+`* * *` | user | find a person by name| view the contact of the person
```

Comment:
Does this repeat the given point below: ` * * * | user | find a person by name | locate details of persons without having to go through the entire list`?